src/button/template/componentTemplate.jsx

```diff
@@ -157,20 +149,31 @@ function renderContent(text : string, { label, locale, color, branding, logoColo
 
         separator() : JsxHTMLNode {
             return <span class={ CLASS.SEPARATOR }></span>;
+        },
+
+        break(value : string) : JsxHTMLNode {
+            let className = `${ CLASS.TEXT }`;
+            return <span class={ className }>{ value.split('<br>')[0] }<br>{ value.split('<br>')[1] }</br></span>;
```

Comment:
I would like to split it dynamically here but can't think of a proper way.

Reply:
I think we can just `return <br />` here and simplify this a lot. The jsxRender function should take care of placing the break in the right place in the content.

Reply:
Resolved.

Reply:
👍